go/file_manager/pfsserver/handler.go

```diff
@@ -0,0 +1,63 @@
+package pfsserver
+
+import (
+	"encoding/json"
+	"github.com/cloud/go/file_manager/pfscommon"
```

Comment:
Are there really benefit of having so much packages :) (pfscommon, pfsmodules), why not just have one package?